pkg/kritis/review/validating_transport.go

```diff
@@ -69,13 +69,17 @@ func (avt *AttestorValidatingTransport) GetValidatedAttestations(image string) (
 		return nil, err
 	}
 	for _, a := range attestations {
-		decoded_sig, err := base64.StdEncoding.DecodeString(a.Signature)
+		// TODO(acamadeo): Temporarily hardcoding this to work only for PGP Attestations.
+		signature := a.PGPAttestation.Signature.Signature
+		decodedSig := make([]byte, len(signature))
```

Comment:
len(signature) seems off -- decoding does not preserve length
i'd just drop the length

Reply:
The slice must have at least a certain length because `StdEncoding.Decode` indexes inside it. I found some examples and have modified this to set the appropriate length.

Reply:
If you revert to the old code, then this can be a one liner and you don't have to worry about the length:
signature, err := base64.StdEncoding.Decode(a.PGPAttestation.Signature.Signature)

Maybe there's something I'm missing here, but I'd suggest just using this one-liner.

Reply:
Done.